client/extensions/woocommerce/app/settings/email/email-settings/components/internal-notification.js

```diff
@@ -61,14 +61,14 @@ const InternalNotification = ( {
 			<ListItemField className="components__notification-component-input">
 				<FormTextInput
 					className={ isPlaceholder ? 'components__is-placeholder' : null }
-					isError={ emailValidationError }
+					isError={ checked && emailValidationError }
 					name={ item.field }
 					onChange={ change }
 					value={ recipient }
-					placeholder={ placeholder }
+					placeholder={ isPlaceholder ? '' : translate( 'Recipient email address.' ) }
```

Comment:
Update this to `Recipient email address(es).` per https://github.com/Automattic/wp-calypso/pull/21648#issuecomment-359568865